architecture/standards/0007-java-pre-requisite.md

```diff
@@ -0,0 +1,53 @@
+# ADR-0007 - Gradle and Java as a pre-requisite
```

Comment:
🤔 I don't think this title is very clear.  As a pre-requisite to what?  Is Gradle and Java a pre-requisite for something?  Or is this talking about Gradle, and how Gradle uses Java as a pre-requisite for something?